packages/@aws-cdk/core/lib/cfn-output.ts

```diff
@@ -60,61 +60,57 @@ export class CfnOutput extends CfnElement {
   }
 
   /**
-   * Returns the description of this Output
+   * A String type that describes the output value.
+   * The description can be a maximum of 4 K in length.
+   *
+   * @default - No description.
    */
   public get description() {
     return this._description;
   }
 
-  /**
-   * Sets this output's description to the parameter
-   * @param description the description to update this Output's description to
-   */
   public set description(description: string | undefined) {
     this._description = description;
   }
 
   /**
-   * Returns the value of this Output
+   * The value of the property returned by the aws cloudformation describe-stacks command.
```

Comment:
Sorry, what does this mean? :)

Reply:
That comment is taken from the props. So the description of the prop's value field and the CfnOutput's value field are the same

Reply:
Fine :)